Compilation error with pop_os! 21.10

Error message:

```
In file included from /usr/include/signal.h:328,
                 from /home/lowin/Documents/projects/univ/PentaCode/Radium-Engine/tests/external/Catch2/src/Catch2_download/single_include/catch2/catch.hpp:8031,
                 from /home/lowin/Documents/projects/univ/PentaCode/Radium-Engine/tests/unittest/unittest.cpp:2:
/home/lowin/Documents/projects/univ/PentaCode/Radium-Engine/tests/external/Catch2/src/Catch2_download/single_include/catch2/catch.hpp:10815:58: error: call to non-‘constexpr’ function ‘long int sysconf(int)’
10815 |     static constexpr std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
      |                                                          ^~~~~~~~~~~
In file included from /usr/include/x86_64-linux-gnu/bits/sigstksz.h:24,
                 from /usr/include/signal.h:328,
                 from /home/lowin/Documents/projects/univ/PentaCode/Radium-Engine/tests/external/Catch2/src/Catch2_download/single_include/catch2/catch.hpp:8031,
                 from /home/lowin/Documents/projects/univ/PentaCode/Radium-Engine/tests/unittest/unittest.cpp:2:
/usr/include/unistd.h:640:17: note: ‘long int sysconf(int)’ declared here
  640 | extern long int sysconf (int __name) __THROW;
      |                 ^~~~~~~
In file included from /home/lowin/Documents/projects/univ/PentaCode/Radium-Engine/tests/unittest/unittest.cpp:2:
/home/lowin/Documents/projects/univ/PentaCode/Radium-Engine/tests/external/Catch2/src/Catch2_download/single_include/catch2/catch.hpp:10874:45: error: size of array ‘altStackMem’ is not an integral constant-expression
10874 |     char FatalConditionHandler::altStackMem[sigStackSize] = {};
````

--- a/tests/external/CMakeLists.txt
+++ b/tests/external/CMakeLists.txt
@@ -1,5 +1,5 @@
 add_custom_target(update)
-external_download_now_git(Catch2 https://github.com/catchorg/Catch2.git v2.12.4)
+external_download_now_git(Catch2 https://github.com/catchorg/Catch2.git v2.13.8)
 
 set(CATCH_USE_VALGRIND OFF CACHE BOOL "Perform SelfTests with Valgrind")
 set(CATCH_BUILD_TESTING OFF CACHE BOOL "Build SelfTest project")

